AndroidAnnotations/androidannotations/src/main/java/org/androidannotations/holder/EComponentHolder.java

```diff
@@ -76,6 +77,10 @@ public JBlock getInitBody() {
 		return getInit().body();
 	}
 
+	public JBlock getIntentFilterTarget(IntentFilterData intentFilterData) {
```

Comment:
`getIntentFilterTarget` should be a method of `HasReceiverRegistration`.
`EComponentHolder` does not implement this interface so the method should not be there.
Write this method in both `EActivityHolder`, `EServiceHolder` and `EFragmentHolder`. Even if the implementation is the same for `EActivityHolder` and `EServiceHolder`.